ios/FluentUI.Demo/FluentUI.Demo.xcodeproj/xcshareddata/xcschemes/Demo.Development.xcscheme

```diff
@@ -75,8 +75,7 @@
             </BuildableReference>
          </TestableReference>
          <TestableReference
-            skipped = "NO"
-            parallelizable = "YES">
```

Comment:
Just to check, removing parallelizable here was what fixed the tests? I wonder why this got added to iOS and not macOS

Reply:
Yep! This was auto added when I first created the launch tests so I think there's a flag set somewhere but I'm looking into it.

Reply:
Hm looking at the macOS scheme, it is also set to parallelizable so I'm wondering why that's not an issue there.